internal/server/v1/triples/golden/bulk_triples_in/result2.json

```diff
@@ -4017,6 +4017,54 @@
       "triples": {
         "measurementDenominator": {
           "nodes": [
+            {
```

Comment:
I think the token here https://github.com/datacommonsorg/mixer/blob/a91ea1008f050776dfec7fabe9d502df1c59e233/internal/server/v1/triples/golden/bulk_triples_in_test.go#L51 does not match the one in https://github.com/datacommonsorg/mixer/blob/a91ea1008f050776dfec7fabe9d502df1c59e233/internal/server/v1/triples/golden/bulk_triples_in/result1.json.

can you update the token in test.go to match the actual token? We have a few similar places, where you can search for "H4sIAAAAAAAA" in`*test.go` and make sure they all match the actual one from `*1.json` file

Reply:
Updated the token in script and regenerated the goldens.